modular_skyrat/modules/automapper/code/automapper_subsystem.dm

```diff
@@ -70,12 +72,47 @@ SUBSYSTEM_DEF(automapper)
 	if(!islist(map_names))
 		map_names = list(map_names)
 	for(var/datum/map_template/automap_template/iterating_template as anything in preloaded_map_templates)
-		if(!(iterating_template.override_map_name in map_names))
+		if(iterating_template.affects_builtin_map && ((SSmapping.config.map_file in map_names) || SSmapping.config.map_file == map_names))
+			// CentComm already started loading objects, place them in the netherzone
+			for(var/turf/old_turf as anything in iterating_template.get_affected_turfs(iterating_template.load_turf, FALSE))
+				init_contents(old_turf)
+		else if(!(iterating_template.required_map in map_names))
 			continue
 		if(iterating_template.load(iterating_template.load_turf, FALSE))
-			add_startup_message("AUTOMAPPER: Successfully loaded map template [iterating_template.name] at [iterating_template.load_turf.x], [iterating_template.load_turf.y], [iterating_template.load_turf.z]!")
+			add_startup_message("Loaded [iterating_template.name] at [iterating_template.load_turf.x], [iterating_template.load_turf.y], [iterating_template.load_turf.z]!")
 			log_world("AUTOMAPPER: Successfully loaded map template [iterating_template.name] at [iterating_template.load_turf.x], [iterating_template.load_turf.y], [iterating_template.load_turf.z]!")
 
+/**
+ * CentCom atoms aren't initialized but already exist, so must be properly initialized and then qdel'd.
+ * Arguments:
+ * * parent - parent turf
+ */
+/datum/controller/subsystem/automapper/proc/init_contents(atom/parent)
+	var/static/list/mapload_args = list(TRUE)
+	// Don't even initialize things in this list. Very specific edge cases.
```

Comment:
why

Reply:
The things in that list either cause irreversible game state changes or randomly spawn things into the void if deleted without the game ticking, for some reason.

Reply:
I don't like it, this should be done in a way where the objects are just never loaded

Reply:
I guess so, but this will probably cause issues down the line

Reply:
I'll be honest, it might. This isn't fullproof by any means, but we can always temporarily cut automapper tson entries for CentCom if it's real bad.